lib/csdl2markdown.js

```diff
@@ -335,14 +336,53 @@ module.exports.csdl2markdown = function (filename, csdl) {
                 + '|' + descriptionInTable(p));
         });
 
-        const style = parent ? '*' : '';
-        (type[voc.Validation.ApplicableTerms] || []).forEach(term => {
-            lines.push(`${style}@${term}${style}|${typeLink({ $Type: term })}|Annotation`);
+        return lines;
+    }
+
+    
+     /**
+     * Table of applicable terms of structured type
+     * @param {object} type Structured type
+     * @return {Array} Array of strings containing Markdown lines
+     */
+    function applicableTermsTable(type) {
+        const lines = [];
+
+        const tLines = applicableTermLines(type);
+        if (tLines.length > 0) {
+            lines.push('');
+            lines.push('**Notable terms:**');
```

Comment:
```suggestion
            lines.push('**Applicable Terms:**');
```

One last change: as this list is the contents of term "ApplicableTerms", why not use that as the headline.